dev/how-to/deploy/tidb-binlog.md

```diff
@@ -331,9 +333,35 @@ The following part shows how to use Pump and Drainer based on the nodes above.
         # the address of the PD cluster nodes
         pd-urls = "http://192.168.0.16:2379,http://192.168.0.15:2379,http://192.168.0.14:2379"
 
+        # [security]
+        # This section is generally annotated if no special security settings are required.
+        # The file path containing a list of trusted SSL CAs connected to the cluster
+        # ssl-ca = "/path/to/ca.pem"
+        # The path to the X509 certificate in PEM form that is connected to the cluster
+        # ssl-cert = "/path/to/drainer.pem"
+        # The path to the X509 key in PEM form that is connected to the cluster
+        # ssl-key = "/path/to/drainer-key.pem"
+
         # [storage]
         # Set to true (by default) to guarantee reliability by ensuring binlog data is flushed to the disk
         # sync-log = true
+
+        # When the available disk capacity is less than the set value, Pump stops writing data.
+        # 42 MB -> 42000000, 42 mib -> 44040192
+        # default: 10 gib
+        # stop-write-at-available-space = "10 gib"
+        # The LSM DB settings embedded in Pump. Unless you know this part well, it is usually annotated.
```

Comment:
**Note:** "注释掉" means "comment out" so that it won't take effect. "annotate" cannot convey this meaning.

```suggestion
        # The LSM DB settings embedded in Pump. Unless you know this part well, it is usually commented out.
```